lib/amazon_ssa_support/instance_metadata.rb

```diff
@@ -0,0 +1,44 @@
+require 'httpclient'
+
+module AmazonSsaSupport
+  class InstanceMetadata
+    
+    def initialize(version='latest')
+      @baseUrl    = 'http://169.254.169.254/'
+      @version    = version
```

Comment:
Any reason to make version an instance variable?  The one set here is only used in the next line, and the one set by "version=" is only used in that method.